docs/src/network-definition-spec.md

```diff
@@ -32,6 +32,7 @@ The network config can be provided both in `json` or `toml` format and each sect
   - `local_path`: string;
   - `remote_name`: string;
 - `default_resources`: (Object) **Only** available in `kubernetes`, represent the resources `limits`/`reservations` needed by the nodes by default.
+- `chain_spec_modifier_commands`: (Array of arrays of strings, optional) Set of commands and their arguments to modify the resulting chain spec before the chain is launched with it. In the arguments, `${chainSpec}`, `${plainSpec}`, and `${plainChainSpec}` will be substituted for the plain spec path and run before the raw chain spec is generated, and `${rawChainSpec}` and `${rawSpec}` will be substituted for the raw chain spec path.
```

Comment:
Based on [this comment](https://github.com/paritytech/zombienet/pull/909/files#r1161537943) I would suggest to alter the `Array of array of strings` to a more descriptive way of explaining what is what in the param